company_multi_logo/README.rst

```diff
@@ -0,0 +1,56 @@
+Company Multiple Logo
+=====================
+
+This module allows to display different logos in reports for a single company.
+For each partner, a logo can be selected to be displayed in the reports for this partner.
+
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+ * Install the module, and then modify the Qweb template report.external_layout_header to display the logo.
```

Comment:
Exactly, that's what I did 2 lines below: e.g. <img t-att-src="'data:image/png;base64,%s' % o.partner_id.get_company_logo(company)"/>